### DISCUSSION TITLE
If the factor is not in known_covariates_names, will TimeSeriesPredictor use it for calculation?

### DISCUSSION THREAD

**Discussion by codezealot** [autogluon/autogluon#4468] [Q&A]
In TimeSeriesPredictor, if the data column is not written in known_covariates_names, will other columns except item_id, target, and timestamps columns not be involved in the calculation?

**Comment by shchur** [comment]
Currently only the TFT model supports covariates that are known only in the past (https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-model-zoo.html#additional-features). This means, other models will ignore them during the forecast. We hope to change that in a future release and add more models that can natively handle past-only covariates.
